sources/Interop/DCommon/um/dcommon/D2D_POINT_2U.Manual.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from um/d2d1helper.h in the Windows SDK for Windows 10.0.18362.0
+// Original source is Copyright © Microsoft. All rights reserved.
+
+using System;
+
+namespace TerraFX.Interop
+{
+    public partial struct D2D_POINT_2U : IEquatable<D2D_POINT_2U>
+    {
+        public static readonly D2D_POINT_2U DEFAULT = new D2D_POINT_2U(0, 0);
```

Comment:
nit: It's probably not worth defining `DEFAULT` when:
* The header file doesn't itself define an explicit default
-and
* The constructor parameter defaults are the same as `new T()`